modules/installation-create-ingress-dns-records.adoc

```diff
@@ -6,7 +6,7 @@
 [id="installation-create-ingress-dns-records_{context}"]
 = Creating the Ingress DNS Records
 
-If you removed the DNS Zone configuration, manually create DNS records that point to the Ingress load balancer.
+If you removed the DNS Zone configuration, or if you enabled the cluster-wide proxy, manually create DNS records that point to the Ingress load balancer.
```

Comment:
s/Ingress load balancer/hostname of the router load balancer service/. You may want to provide an example on how to get the Ingress load-balancer. Here's an example for the default router:
```
$ oc get svc/router-default -n openshift-ingress -o yaml | grep hostname
    - hostname: a4a37c40e0af011eabdc302eee56a296-792674788.us-west-2.elb.amazonaws.com
```

Reply:
@danehans What would someone need to know the host name for? (I don't see anything in this procedure that uses it).